prdoc/pr_3761.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Snowbridge Ethereum client improvements and prevent relayer rewards from message processing to fail
```

Comment:
@claravanstaden iirc, this title needs to match PR title, so please better change one of them. @EgorPopelyaev right?

Reply:
No, it doesn't need to match. E.g. I just merged https://github.com/paritytech/polkadot-sdk/pull/3927 with prdoc with slightly different title than PR.

Reason it's failing is missing `bump` qualifier. See e.g. https://github.com/acatangiu/polkadot-sdk/blob/67e322206ea9d1990baae560c73a46e72dff5b2d/prdoc/pr_3930.prdoc#L10

Reply:
Updated prdoc over here: https://github.com/paritytech/polkadot-sdk/pull/3761/commits/ef97c0bbc3d6bf5607fb2112bc6d5d4ad4d11c25 I marked the Snowbridge crate version bumps as major because this is a major, and breaking change (from an interface perspective).

Reply:
Fixed :)